ocs_ci/ocs/platform_nodes.py

```diff
@@ -1286,25 +1286,26 @@ def get_rhcos_worker_template(self):
             path (str): local path to template file
 
         """
-        common_base = 'v3-launch-templates/functionality-testing'
+        common_base = 'functionality-testing'
         ocp_version = get_ocp_version('_')
         relative_template_path = os.path.join(
             f'aos-{ocp_version}',
             'hosts/upi_on_aws-cloudformation-templates'
         )
 
-        template_url = os.path.join(
-            f'{constants.OCP_QE_MISC_REPO}',
-            'plain',
+        path_to_file = os.path.join(
             f'{common_base}',
             f'{relative_template_path}',
             f'{constants.AWS_WORKER_NODE_TEMPLATE}'
         )
-        logger.info(f"Getting template from url {template_url}")
+        logger.info(
+            f"Getting file '{path_to_file}' from "
+            f"git repository {constants.OCP_QE_MISC_REPO}"
+        )
         tmp_file = os.path.join(
             '/tmp', constants.AWS_WORKER_NODE_TEMPLATE
         )
-        download_file(template_url, tmp_file)
+        download_file_from_git_repo(constants.OCP_QE_MISC_REPO, path_to_file, tmp_file)
```

Comment:
@vavuthu do you know if we need different branches for different versions?

Reply:
appropriate branch is getting from https://github.com/red-hat-storage/ocs-ci/pull/2849/files#diff-da5af5766904307b6dcbf4c4ee2a04d9R1292

Reply:
different branch is not needed since different ocp version is maintaining in folder